.gitignore

```diff
@@ -192,3 +192,4 @@ nbproject/*
 *.cd
 a.out
 cmake-build-debug/*
+.ccls-cache/*
```

Comment:
What does this do, and can we do without it?  I'd rather get a clean "add calendar" build, and leave everything else (such as build changes) to a separate PR.

Reply:
It's the cache folder from [ccls](https://github.com/MaskRay/ccls) which I'm using in Emacs.

Reply:
Any harm if I remove it prior to pushing to master?

Reply:
Not at all.